package.json

```diff
@@ -2,8 +2,11 @@
   "name": "@ably/features",
   "version": "1.2.0-alpha.1",
   "scripts": {
+    "dev": "concurrently \"npm run build:dev\" \"live-server output\"",
```

Comment:
Better to align with the idiomatic conventions established by projects like React and rename `dev` to `start`... then your users need only use `npm start` instead of `npm run dev`. 😁